src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.xml

```diff
@@ -29,6 +29,10 @@
         </service>
         <service id="Symfony\Component\Messenger\Transport\Serialization\SerializerInterface" alias="messenger.default_serializer" />
 
+        <service id="serializer.normalizer.flatten_exception" class="Symfony\Component\Messenger\Transport\Serialization\Normalizer\FlattenExceptionNormalizer">
+            <tag name="serializer.normalizer" priority="-880" />
+        </service>
```

Comment:
I didn't realize that it was on 4.4. As this is a new feature, it should be moved to master instead, where the configuration should be moved to the new PHP config file.

Reply:
I considered this as a fix, because this breaks one of the Messenger commands, but I can move it to master if you consider this to be a feature.

Reply:
This is a new feature as we are adding a class and changing behavior.